### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by rihabbanday**
Add Scot Steele as RC2 owner

### CODE REVIEW

**Review by pgoyal01 — APPROVED** [review]

**Review by michaelspedersen — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -93,15 +93,15 @@
...
-/doc/ref_cert/RC2/chapters/chapter01.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter02.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter03.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter05.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter04.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter06.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter07.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter08.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter09.md        @walterkozlowski 
+/doc/ref_cert/RC2/chapters/chapter01.md        @walterkozlowski @ss8171
```
Comment (pgoyal01) [inline CODEOWNERS:96]:
```suggestion
/doc/ref_cert/RC2/chapters/chapter01.md        @walterkozlowski @steelescotr
```

File: CODEOWNERS [hunk 2]
```diff
@@ -93,15 +93,15 @@
...
+/doc/ref_cert/RC2/chapters/chapter02.md        @walterkozlowski @ss8171
+/doc/ref_cert/RC2/chapters/chapter03.md        @walterkozlowski @ss8171
```
Comment (pgoyal01) [inline CODEOWNERS:98]:
```suggestion
/doc/ref_cert/RC2/chapters/chapter03.md        @walterkozlowski @steelescotr
```

File: CODEOWNERS [hunk 3]
```diff
@@ -93,15 +93,15 @@
...
+/doc/ref_cert/RC2/chapters/chapter05.md        @walterkozlowski @ss8171
```
Comment (pgoyal01) [inline CODEOWNERS:99]:
```suggestion
/doc/ref_cert/RC2/chapters/chapter05.md        @walterkozlowski @steelescotr
```

File: CODEOWNERS [hunk 4]
```diff
@@ -93,15 +93,15 @@
...
+/doc/ref_cert/RC2/chapters/chapter04.md        @walterkozlowski @ss8171
```
Comment (pgoyal01) [inline CODEOWNERS:100]:
```suggestion
/doc/ref_cert/RC2/chapters/chapter04.md        @walterkozlowski @steelescotr
```

File: CODEOWNERS [hunk 5]
```diff
@@ -93,15 +93,15 @@
...
+/doc/ref_cert/RC2/chapters/chapter06.md        @walterkozlowski @ss8171
```
Comment (pgoyal01) [inline CODEOWNERS:101]:
```suggestion
/doc/ref_cert/RC2/chapters/chapter06.md        @walterkozlowski @steelescotr
```

File: CODEOWNERS [hunk 6]
```diff
@@ -93,15 +93,15 @@
...
+/doc/ref_cert/RC2/chapters/chapter07.md        @walterkozlowski @ss8171
```
Comment (pgoyal01) [inline CODEOWNERS:102]:
```suggestion
/doc/ref_cert/RC2/chapters/chapter07.md        @walterkozlowski @steelescotr
```

File: CODEOWNERS [hunk 7]
```diff
@@ -93,15 +93,15 @@
...
+/doc/ref_cert/RC2/chapters/chapter08.md        @walterkozlowski @ss8171
```
Comment (pgoyal01) [inline CODEOWNERS:103]:
```suggestion
/doc/ref_cert/RC2/chapters/chapter08.md        @walterkozlowski @steelescotr
```

File: CODEOWNERS [hunk 8]
```diff
@@ -93,15 +93,15 @@
...
+/doc/ref_cert/RC2/chapters/chapter09.md        @walterkozlowski @ss8171
```
Comment (pgoyal01) [inline CODEOWNERS:104]:
```suggestion
/doc/ref_cert/RC2/chapters/chapter09.md        @walterkozlowski @steelescotr
```

**Review by pgoyal01 — APPROVED** [review]

File: CODEOWNERS [hunk 9]
```diff
@@ -93,15 +93,15 @@
...
-/doc/ref_cert/RC2/chapters/chapter09.md        @walterkozlowski 
+/doc/ref_cert/RC2/chapters/chapter01.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter02.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter03.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter05.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter04.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter06.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter07.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter08.md        @walterkozlowski @steelescotr
+/doc/ref_cert/RC2/chapters/chapter09.md        @walterkozlowski @steelescotr
```
Comment (gkunz) [inline CODEOWNERS:104]: How about simplifying this like the following:

```suggestion
/doc/ref_cert/RC2/        @walterkozlowski @steelescotr
```

**Review by gkunz — CHANGES_REQUESTED** [review]

**Comment by pgoyal01** [comment]
@gkunz should all other entries be similarly changed?

**Comment by gkunz** [comment]
> @gkunz should all other entries be similarly changed?

I think that would be a good idea.

**Comment by pgoyal01** [comment]
@gkunz Done - Thanks

File: CODEOWNERS [hunk 10]
```diff
@@ -8,103 +8,31 @@
...
-/doc/ref_cert/RC2/chapters/chapter06.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter07.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter08.md        @walterkozlowski
-/doc/ref_cert/RC2/chapters/chapter09.md        @walterkozlowski 
+/doc/ref_cert/RC2/                   @walterkozlowski @steelescotr
 
-# Edge
-/doc/tech/usecases.md        @ASawwaf @walterkozlowski
+<!--- # Edge
+/doc/tech/usecases.md        @ASawwaf @walterkozlowski --->
```
Comment (gkunz) [inline CODEOWNERS:35]: I believe the correct character for comments is '#'

```suggestion
# Edge workstream
# /doc/tech/usecases.md              @ASawwaf @walterkozlowski
```

**Review by gkunz — CHANGES_REQUESTED** [review]
Thanks for updating the other sections as well. There is one syntax issue left.

**Review by gkunz — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Pull request merged**